web/views.py

```
@@ -6929,3 +6933,207 @@ def delete_post(request, post_id):
     if request.method == "POST":
         post.delete()
     return redirect("social_media_dashboard")
+
+
+@login_required
+def study_planner(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `study_planner`

(ANN201)